### PR TITLE
Fix package dependencies for security vulnerabilities

### PR DESCRIPTION
**Pull request by shirayu**
After publish of ``nlcst-pattern-match``, ``nlcst-pattern-match": "^1.3.5",`` in ``packages/match-test-replace/package.json`` should be updated.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2,185 +2,1224 @@
...
+  version "5.2.7"
+  resolved "https://registry.yarnpkg.com/@types/mocha/-/mocha-5.2.7.tgz#315d570ccb56c53452ff8638738df60726d5b6ea"
+  integrity sha512-NYrtPht0wGzhwe9+/idPaBB+TqkY9AhTvOLMkThm0IoEfLaiVQZwBwyJ5puCkO3AUCWrmcoePjp2mbFocKy4SQ==
+
+"@types/node@*", "@types/node@>= 8":
+  version "14.14.20"
+  resolved "https://registry.yarnpkg.com/@types/node/-/node-14.14.20.tgz#f7974863edd21d1f8a494a73e8e2b3658615c340"
+  integrity sha512-Y93R97Ouif9JEOWPIUyU+eyIdyRqQR0I8Ez1dzku4hDx34NWh4HbtIc3WNzwB1Y9ULvNGeu5B8h8bVL5cAk4/A==
 
 "@types/node@^10.3.4":
```
Comment (azu) [inline yarn.lock:958]: - [Build #86 - azu/nlp-pattern-match - Travis CI](https://travis-ci.com/github/azu/nlp-pattern-match/builds/212294928)
  - [v14.6.3 has error in node_modules/@types/node/index.d.ts(20,1): error TS1084: Invalid 'reference' directive syntax. · Issue #47250 · DefinitelyTyped/DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/47250)

@types/node updating cause this error?
Probably, you need to typescript version.

Comment (shirayu) [reply yarn.lock:958]: Thank you, I fixed it.

File: package.json [hunk 2]
```diff
@@ -36,8 +36,9 @@
   },
   "homepage": "https://github.com/azu/nlp-pattern-match",
   "devDependencies": {
+    "@types/node": "14.6.2",
```
Comment (azu) [inline package.json]: I recommened that just update typescript or use `resolusions` field.

- [Selective dependency resolutions | Yarn](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/)

File: package.json [hunk 3]
```diff
@@ -37,9 +37,10 @@
   "homepage": "https://github.com/azu/nlp-pattern-match",
   "devDependencies": {
     "husky": "^0.14.3",
-    "lerna": "^2.11.0",
+    "lerna": "^3.22.1",
     "lint-staged": "^7.2.0",
-    "prettier": "^1.13.5"
+    "prettier": "^1.13.5",
+    "typescript": "4.1.3"
```
Comment (azu) [inline package.json:43]: Does this install multiple typescript versions into the project?
It seems that yarn.lock has typescript 4.1.3 and typescript@^2.5.3.(These are not deduped)
Probably, you need to update each `typescript`s via `yarn upgrade-interactive --latest`.

Comment (shirayu) [reply package.json:43]: Yes, I updated it.

**Comment by azu** [comment]
Thanks!

**Pull request merged**